src/components/Timer/index.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Flex, Heading, Text } from '@pancakeswap-libs/uikit'
+import TimerTooltip from './TimerTooltip'
+
+export interface TimerProps {
+  timerStage?: string
+  minutes?: number
+  hours?: number
+  days?: number
+  showTooltip?: boolean
+  blockNumber?: number
+  HeadingTextComponent?: any
+  BodyTextComponent?: any
```

Comment:
`ReactElement` is what I think you need here...

Reply:
Dead right. I left this in by accident - was experimenting with ways of passing a component as props it.